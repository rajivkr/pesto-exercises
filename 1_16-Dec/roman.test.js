const romanize = require('./roman').getRomanMemoized;
describe('convert to roman test cases', () => {
  it('should convert given num to roman letter', () => {
    expect(romanize(5)).toMatchSnapshot();
    expect(romanize(500)).toMatchSnapshot();
  });
});
