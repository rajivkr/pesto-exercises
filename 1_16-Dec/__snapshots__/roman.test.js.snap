// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`convert to roman test cases should convert given num to roman letter 1`] = `"V"`;

exports[`convert to roman test cases should convert given num to roman letter 2`] = `"D"`;
